pagopa/io-app
# fix: [IOCOM-737] Update the display rule of the RemoteContentBanner component

## Short description
This PR updates the display rule of the RemoteContentBanner component. In order to see the banner, the `has_remote_content` property must be true.

<details open><summary>Details</summary>
<p>

| without banner | with banner |
| - | - |
| <video src="https://github.com/pagopa/io-app/assets/29163287/3d7d7953-f490-416e-943f-a0b128d6c6fa" /> | <video src="https://github.com/pagopa/io-app/assets/29163287/447438b6-691f-4246-8d80-b9e5ef092de7" /> | 

</p>
</details> 

## List of changes proposed in this pull request
- updated specs
- added the `hasRemoteContent` property to the `UIMessageDetails` type
- updated the transformer `toUIMessageDetails` in order to return the `hasRemoteContent` property
- updated the `MessageDetail` component: the banner will be displayed iff the `hasRemoteContent` property is true

## How to test
Using `io-dev-api-server`, generate a message with remote content. Open the message detail and check that the banner is displayed correctly.
